packages/php-wasm/compile/build-assets/php_wasm.c

```diff
@@ -200,7 +200,7 @@ const char WASM_HARDCODED_INI[] =
 	"always_populate_raw_post_data = -1\n"
 	"upload_max_filesize = 2000M\n"
 	"post_max_size = 2000M\n"
-	"disable_functions = proc_open,popen,curl_exec,curl_multi_exec\n"
+	"disable_functions = proc_open,popen,curl_exec,curl_multi_exec,shell_exec,proc_close\n"
```

Comment:
```suggestion
	"disable_functions = proc_open,popen,curl_exec,curl_multi_exec,proc_close\n"
```

It's possible to provide a custom JavaScript handler for `shell_exec` so let's soft-disable it in php.ini. This way it's possible to re-enable it in runtime.

Reply:
Is the same possible also for `proc_open`? Overriding that one would be needed for making Behat tests work.

Reply:
Not at the moment. `shell_exec` was easier to handle because it returns the output. `proc_open` returns a resource and exposes low-level process calls and will require more effort. Definitely possible to support, though.